gremlin-util/src/test/java/org/apache/tinkerpop/gremlin/util/ser/GraphSONMessageSerializerV3Test.java

```diff
@@ -288,9 +288,9 @@ public void shouldSerializeToTreeJson() throws Exception {
         Iterator<Vertex> vertexKeys = firstTree.keySet().iterator();
 
         Map expectedValues = new HashMap<Integer, String>() {{
-            put(3, "vadas");
-            put(5, "lop");
-            put(7, "josh");
+            put(2, "vadas");
```

Comment:
what made all ids change (i see you had to do this in other tests as well)?

Reply:
the main reason is the change in the processing order of the vertices. Because of this, the vertex properties were created in a different order and received different IDs. Property IDs are not defined anywhere so I left it as is